packs/pathfinder-bestiary/brass-dragon-adult.json

```
@@ -461,7 +461,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The brass dragon breathes fire in a @Template[line|distance:80]{80-foot line} that deals @Damage[12d6[fire]] damage (@Check[reflex|dc:32|basic] save).</p>\n<p>The dragon can't use Breath Weapon again for [[/gmr 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>"
+                    "value": "<p>The brass dragon breathes fire in a @Template[line|distance:80] that deals @Damage[12d6[fire]] damage (@Check[reflex|dc:32|basic|option:damaging-effect] save).</p>\n<p>The dragon can't use Breath Weapon again for [[/gmr 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>"
```

```suggestion
                    "value": "<p>The brass dragon breathes fire in a @Template[line|distance:80] that deals @Damage[12d6[fire]] damage (@Check[reflex|dc:32|basic] save).</p>\n<p>The dragon can't use Breath Weapon again for [[/gmr 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>"
```
not needed for basic saves